Tests/Site/IdentifierMappingTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+declare(strict_types=1);
+
+namespace FH\Bundle\MultiSiteBundle\Tests\Site;
+
+use FH\Bundle\MultiSiteBundle\Site\IdentifierMapping;
+use PHPUnit\Framework\TestCase;
+
+final class IdentifierMappingTest extends TestCase
+{
+    private const MAPPING = [
+        'amsterdam' => [
+            'hostnames' => [
+                'amsterdam.local',
+                'amsterdam.nl'
+            ],
+            'locales' => ['nl', 'en']
+        ],
+        'tilburg' => [
+            'hostnames' => [
+                'tilburg.local',
+                'tilburg.nl'
+            ],
+            'locales' => ['nl']
+        ],
+        'den-bosch' => [
+            'hostnames' => [
+                'denbosch.local',
```

Comment:
Je zou in theorie hier ook `den-bosch.local` etc kunnen gebruiken, meer in lijn?